numpy/core/src/multiarray/conversion_utils.c

```diff
@@ -419,6 +419,9 @@ PyArray_SortkindConverter(PyObject *obj, NPY_SORTKIND *sortkind)
     else if (str[0] == 'm' || str[0] == 'M') {
         *sortkind = NPY_MERGESORT;
     }
+    else if (str[0] == 's' || str[0] == 'S') {
+        *sortkind = NPY_STABLESORT;
```

Comment:
we don't really need a new enum entry, just have stable map to `NPY_MERGESORT?

Reply:
though for the C-api it is probably useful

Reply:
Yes, you are right, I could have done this without adding an additional enum but I did this keeping in mind the discussion we had on the issue

> I think this is a great idea, as long as we don't document it as specifically being an alias for mergesort, but as providing a default stable sorting algorithm: that way if we one day implement, say count sort for uint8, we can make stable point to that algorithm for that dtype.

I guess if something like this is done in a future patch then it would require creating an enum.